packages/front-end/components/Experiment/BreakDownResults.tsx

```diff
@@ -19,8 +20,8 @@ import UsersTable from "./UsersTable";
 
 const FULL_STATS_LIMIT = 5;
 
-type TableDef = {
-  metric: MetricInterface;
+export type TableDef = {
+  metric?: MetricInterface;
```

Comment:
I couldn't figure out from the PR what the optional typing solves

Reply:
(you can see where I use the optional typing here):
https://github.com/growthbook/growthbook/pull/1193/files#diff-0edd6e7dc2561153b4cd40754bee622e23486080cf36ddc3f1656e9629e2a330R99-R102

Reply:
This is redundant now.